lib/voxpupuli/acceptance/spec_helper_acceptance.rb

```diff
@@ -21,9 +21,8 @@ def configure_beaker(modules: :metadata, &block)
       unless %w[none preinstalled].include?(collection)
         BeakerPuppetHelpers::InstallUtils.install_puppet_release_repo_on(host, collection)
       end
-      package_name = ENV.fetch('BEAKER_PUPPET_PACKAGE_NAME',
-                               BeakerPuppetHelpers::InstallUtils.puppet_package_name(host,
-                                                                                     prefer_aio: collection != 'none'))
+      beaker_package_name = BeakerPuppetHelpers::InstallUtils.package_name(host, prefer_aio: collection != 'none', requirement_name: collection.gsub(/\d+/, ''))
```

Comment:
Thinking out loud: would it make sense to pass in `collection` and leave the specific logic to that? I'm not sure I like it very much, but the whole idea of a helper is to move logic there.

Perhaps even introduce a higher level method that I've previously avoided that installs both the release file and package. Like `BeakerPuppetHelpers::InstallUtils.install_on(host, collection)`.

Further extrapolating: do we need some way to indicate a preference to `openvox` vs `puppet`? `collection` won't tell us that if it's `none` or `preinstalled`.